TickSync.py

```diff
@@ -14,10 +14,11 @@
 from SimConfig import *
 from util.Utils import truncate
 import glog as log
+from pynput import keyboard
 
 class TickSync():
 
-    def __init__(self, rate = 30, realtime = True, block = False, verbose = False, label = "", sim_start_time = 0.0):
+    def __init__(self, rate = 30, realtime = True, block = False, verbose = False, label = "", sim_start_time = 0.0, usr_input = False):
```

Comment:
I think no changes are needed to this file at all. You can restore the whole file from master using
```
$ git checkout master -- TickSync.py
$ git add TickSync.py
$ git commit -m "restore from master"
```
Also fix the line 74 in `GSServer.py`